src/main/resources/application.yaml

```diff
@@ -64,3 +90,21 @@ security:
     - "/csrf"
     - "/error"
     - "/favicon.ico"
+
+idam:
+  s2s-auth:
+    totp_secret: ${COMMON_DATA_API_S2S_SECRET:}
```

Comment:
Should not this be `COMMONDATA_API_S2S_SECRET`?

Reply:
Thanks @SujeethKumarAri , Good Catch.
Fixed it.